lib/gstreamer/msdk/transcoder.py

```diff
@@ -22,11 +22,13 @@ class TranscoderTest(BaseTranscoderTest):
         sw = (dict(maxres = (16384, 16384)), have_gst_element("openh264dec"), "h264parse ! openh264dec ! videoconvert"),
         hw = (platform.get_caps("decode", "avc"), have_gst_element("msdkh264dec"), "h264parse ! msdkh264dec"),
         va_hw = (platform.get_caps("decode", "avc"), have_gst_element("vah264dec"), "h264parse ! vah264dec"),
+        d3d_hw = (platform.get_caps("decode", "avc"), have_gst_element("d3d11h264dec"), "h264parse ! d3d11h264dec"),
```

Comment:
Please use the same key (i.e. d3d11_hw) as in https://github.com/intel/vaapi-fits/pull/520.  This will allow for using the same test case config across multiple middleware components.

Reply:
Ok, done. Thanks!